app/helpers/groups_helper.rb

```diff
@@ -36,14 +36,35 @@ def delete_group_link(group, attrs = {})
             }.merge(attrs)
   end
 
+  def kick_member_link(group, member)
+    form_for(
+      :anything,
```

Comment:
What does this refer to? Tried to look up documentation, but couldn't find anything.

Reply:
Should I change this to `form_tag`? Basically this is for the kick member link as I transformed it from `link_to` tag to `form_for` tag to make the `method: :delete` work. (It's basically a hackish solution to a problem we can't understand). So the purpose of `:anything` is just to fill the placeholder model because `form_for` expects a model beforehand. If you try to remove it, it will not work.